.github/workflows/unit-testing.yml

```diff
@@ -3,6 +3,8 @@ name: Unit testing
 on:
   pull_request:
 
+permissions: {}
```

Comment:
I thought we had to actually set read/write?

Reply:
I'm taking my cue from https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs